### ISSUE TITLE
Installing latest version on Magento 2.1.17 fails on a dependency

### ISSUE THREAD

**Issue by BeSite** [tig-nl/postnl-magento2#109]
```
composer require tig/postnl-magento2 1.7.4                                                                                                                                  16:15:12
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - tig/postnl-magento2 1.7.4 requires zendframework/zend-barcode 2.7 -> satisfiable by zendframework/zend-barcode[2.7.0].
    - Installation request for tig/postnl-magento2 1.7.4 -> satisfiable by tig/postnl-magento2[1.7.4].
    - Conclusion: remove zendframework/zend-servicemanager 2.5.1
    - Conclusion: don't install zendframework/zend-servicemanager 2.5.1
    - zendframework/zend-barcode 2.7.0 requires zendframework/zend-servicemanager ^2.7.8 || ^3.3 -> satisfiable by zendframework/zend-servicemanager[2.7.10, 2.7.11, 2.7.8, 2.7.9, 3.3.0, 3.3.1, 3.3.2, 3.4.0].
    - Can only install one of: zendframework/zend-servicemanager[2.7.10, 2.5.1].
    - Can only install one of: zendframework/zend-servicemanager[2.7.11, 2.5.1].
    - Can only install one of: zendframework/zend-servicemanager[2.7.8, 2.5.1].
    - Can only install one of: zendframework/zend-servicemanager[2.7.9, 2.5.1].
    - Can only install one of: zendframework/zend-servicemanager[3.3.0, 2.5.1].
    - Can only install one of: zendframework/zend-servicemanager[3.3.1, 2.5.1].
    - Can only install one of: zendframework/zend-servicemanager[3.3.2, 2.5.1].
    - Can only install one of: zendframework/zend-servicemanager[3.4.0, 2.5.1].
    - Installation request for zendframework/zend-servicemanager (locked at 2.5.1) -> satisfiable by zendframework/zend-servicemanager[2.5.1].
```

I'm trying to install the latest version using composer, and I'm receiving the error above.

I did some digging, and it seems that Magento 2.1.17 requires zend-servicemanager ~2.5.0
```magento/magento2-base              2.1.17  requires  zendframework/zend-servicemanager (~2.5.0)
magento/product-community-edition  2.1.17  requires  zendframework/zend-servicemanager (~2.5.0)```

However, tig/postnl requires zend-barcode 2.7, which on its own requires zend-servicemanager ^2.7.8 || ^3.1

**Comment by tig-daanvandenbergh** [comment]
Hi @BeSite,

Thank you for notifying us of this issue. 

Since Zend_Barcode was dropped from Magento's implementation of Zend Framework 1 in Magento 2.3.x, we need to require this zend/barcode in order to provide compatibility with Magento 2.3.x. 

If upgrading to a later version (Magento 2.2.x or higher) or manual installation to app/code is not an option, please be patient. In our next release we'll include compatibility with more zend/barcode versions to increase compatibility with earlier Magento versions.

**Comment by tig-daanvandenbergh** [comment]
We've just released v1.8.0 of the PostNL-extension adding new features, including compatibility with more zend/barcode versions.

Thanks you for notifying us. I'm closing this issue.
